dts/msm8916/msm8939-mtp.dts

```diff
@@ -12,7 +12,24 @@
 	alcatel-idol3 {
 		model = "Alcatel OneTouch Idol 3 (5.5)";
 		compatible = "alcatel,idol3", "qcom,msm8939", "lk2nd,device";
-		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_r63315_1080p_video:1:none";
+		lk2nd,match-panel;
+
+		panel {
+			compatible = "alcatel,idol3-panel";
+
+			qcom,mdss_dsi_jdi_1080p_video {
+				compatible = "alcatel,idol3-panel-jdi";
+			};
+			qcom,mdss_dsi_nt35596_1080p_video {
+				compatible = "alcatel,idol3-panel-nt35596";
```

Comment:
Do you have this panel? Not sure if we should support it, perhaps this is just some early prototype?

Reply:
My devices have nt35596 (not v03) and r63315, and saw no evidence of others being used.
Only those two have generated drivers... Should the others be removed ?

Reply:
The jdi one looks like an exact copy from the qcom tree, it probably doesn't actually exist. I'd say we can remove that one. Not sure about the v3, you can keep it or remove it. Toss a coin maybe :D

Reply:
Ok I removed the jdi one